trade-finance-manager-api/src/graphql/helpers/amendment.helpers.js

```diff
@@ -0,0 +1,189 @@
+const { format, fromUnixTime } = require('date-fns');
+const { formattedNumber } = require('../../utils/number');
+const { decimalsCount, roundNumber } = require('../../v1/helpers/number');
+const { CURRENCY } = require('../../constants/currency');
+const api = require('../../v1/api');
+const { DEALS } = require('../../constants');
+
+// returns the formatted amendment value and currency (without conversion)
+const amendmentChangeValueExportCurrency = (amendment) => {
+  const { currency, value: amendmentValue } = amendment;
+
+  if (currency && amendmentValue) {
+    return `${currency} ${formattedNumber(amendmentValue)}`;
+  }
+
+  return null;
+};
+
+// calculates new facility value in GBP
+const calculateNewFacilityValue = (exchangeRate, amendment) => {
+  const { currency, value: amendmentValue } = amendment;
+  let newValue;
+
+  if (currency && amendmentValue) {
```

Comment:
Please re-factor this code:
* Nested `if` are not a recommended practice.
* Only `if` inside `else` is not ideal.

Reply:
fixed